src/Core/Scheduler/Providers/WMTS_Provider.js

```diff
@@ -113,6 +113,9 @@ WMTS_Provider.prototype.getColorTexture = function getColorTexture(coordWMTS, la
     return OGCWebServiceHelper.getColorTextureByUrl(url, layer.networkOptions).then((texture) => {
         const result = {};
         result.texture = texture;
+        if (layer.options.mimetype == 'image/png') {
+            texture.premultiplyAlpha = true;
```

Comment:
This is wrong IMHO: you can't deduce `premultiplyAlpha` solely based on the mimetype.

Reply:
what would you expect?

Reply:
You can't guess the value, hence why three default to false.
`premultiplyAlpha` means that the data has been prepared in a specific way - which is quite unlikely in GIS data.

Reply:
the easiest way is to add it as an option to the layer

Reply:
Yes.
Is there a service that would serve premultiplied-alpha image though?

Reply:
So back on this:
- texture.premultiplyAlpha will simply ask WebGL to perform color.rgb *= color.a when loading this texture.
- material.premultipliedAlpha (which you're not using) changes the blending equation to account for premultiplied alpha texture

So if you use only premultiplyAlpha without premultipliedAlpha the rgb channels are going to be multiply twice by alpha:
```
   pixel_color = premultiplied_color * a + pixel_color * (1 - a)
   pixel_color = (original_texture_color * a) * a + pixel_color * (1 - a)
```

Reply:
I will remove the `premultiplyAlpha` because the problem doesn't come from that.
white borders come from Linear Filter


![png2](https://user-images.githubusercontent.com/11291849/32948594-12d9fdc0-cba0-11e7-8b47-9b87374891f6.png)

Reply:
Filtering and premultiplied-alpha are related topics.
Any filtering applied on a texture with transparent pixels (except NEAREST which introduces ugly aliasing) will cause problem - that's why premultiplied alpha has been created in the first place...

Reply:
I change the blending in : 
```js
diffuseColor.rgb = (layerColor.rgb * layerColor.a) + (diffuseColor.rgb * (1.0 - layerColor.a));
```

Reply:
Changing the blending in the shader is a different issue.

As long as you don't use premultipliedAlpha + premultiplyAlpha you'll get blended pixels color when reading the texture from the shader.

Reply:
`material.premultipliedAlpha` is to material blending.
Here, we are talking about  color layers blending.

Reply:
>  Here, we are talking about color layers blending.

I don't think you can really separate the 2 issues.
Anyway, what is your change fixing?

You're replacing:
```glsl
diffuseColor = mix( diffuseColor,layerColor, lum*paramsA.w * layerColor.a);
```
With the equivalent code:
```
layerColor.a *= paramsA.w;
layerColor.a *= lum;
diffuseColor.rgb = (layerColor.rgb * layerColor.a) + (diffuseColor.rgb * (1.0 - layerColor.a));
```

So I don't really expect this to fix anything..

Reply:
oh yes I didn't have to push the trick